packages/cli/src/utils/convert.js

```diff
@@ -40,8 +40,9 @@ const createFile = async (content, filename, dir) => {
   endSpinner();
 };
 
-const prettifyJs = (code) =>
+const prettifyJs = async (code) =>
```

Comment:
prettier.format returns a Promise since v3. This means `renderDefinitionSlice` below which uses `prettifyJs` needs to await it. Interestingly, the callers for `renderDefinitionSlice` itself already awaited the response 👍🏻

The other usage of `prettifyJs` is in `renderIndex` which already returns a Promise.